rwgame/ingamestate.cpp

```diff
@@ -15,12 +15,54 @@
 #include <script/ScriptMachine.hpp>
 #include <dynamics/RaycastCallbacks.hpp>
 
+#include <unordered_map>
+
 constexpr float kAutoLookTime = 2.f;
 constexpr float kAutolookMinVelocity = 0.2f;
 const float kMaxRotationRate = glm::half_pi<float>();
 const float kCameraPitchLimit = glm::quarter_pi<float>() * 0.5f;
 const float kVehicleCameraPitch = glm::half_pi<float>() - glm::quarter_pi<float>() * 0.25f;
 
+// Hardcoded Controls SDLK_* -> GameInputState::Control
+const std::unordered_multimap<int, GameInputState::Control> kDefaultControls = {
+	/* On Foot */
+	{ SDLK_LCTRL,     GameInputState::FireWeapon },
+	{ SDLK_KP_ENTER,  GameInputState::NextWeapon },
+	{ SDLK_KP_PERIOD, GameInputState::LastWeapon },
```

Comment:
Seeing how this maps the original default mapping, shouldn't we also map the alternative buttons here?
(If you don't want to do it now, we should create an issue about it)